qutip/solver/result.py

```diff
@@ -507,26 +572,22 @@ def _add_first_traj(self, trajectory):
 
         self.average_e_data = {
             k: list(avg_expect)
-            for k, avg_expect
-            in zip(self._raw_ops, self._sum_expect)
+            for k, avg_expect in zip(self._raw_ops, self._sum_expect)
         }
-        self.e_data = self.average_e_data
-        if self.options['keep_runs_results']:
+        if self.options["keep_runs_results"]:
             self.runs_e_data = {k: [] for k in self._raw_ops}
-            self.e_data = self.runs_e_data
 
     def _store_trajectory(self, trajectory):
         self.trajectories.append(trajectory)
 
-    def _reduce_states(self, trajectory):
-        self._sum_states = [
+    def _reduce_density_matrix(self, trajectory):
```

Comment:
I read that as reducing density matrix, (dm input), not reducing to density matrix...
This is confusing, could you revert to the original names.